### PR TITLE
Add note about 1.1.0 being yanked

### PR DESCRIPTION
**Pull request by nicoddemus**
Related to #403

### CODE REVIEW

**Review by bluetech — APPROVED** [review]

**Comment by RonnyPfannschmidt** [comment]
We should not close until the actual fix land's

**Review by RonnyPfannschmidt — APPROVED** [review]

**Comment by nicoddemus** [comment]
It has landed already in PyPI, I'm just waiting for confirmation from `conda`.

**Comment by RonnyPfannschmidt** [comment]
@nicoddemus what i mean was the Bugfix for the issue,not the yank

As far as I understood that's incoming tomorrow

**Pull request merged**
